CNAME

```diff
@@ -1 +0,0 @@
-omemo.top
```

Comment:
@ReneVolution But running it on my github was really nice and easy. I wouldn't like to go through some painful setup of server stuff etc. The nice thing with the current workflow is that it's very close to a wiki and at least for me that removes a large threshold of contributing.